__init__.py

```diff
@@ -16,21 +17,21 @@ def __init__(self):
         self.connected = False
         self.wifi_process = None
         self.debug = False  # dev setting, VERY VERBOSE DIALOGS
-        # TODO expose all this in skill settings
-        self.ssid = "OVOS"
+        self.ssid = None
         self.pswd = None
+        self.setup_url = None
         self.grace_period = 45
         self.time_between_checks = 30  # seconds
         self.mycroft_ready = False
-        self.wifi_command = "sudo /usr/local/sbin/wifi-connect --portal-ssid {ssid}"
-        if self.pswd:
-            self.wifi_command += " --portal-passphrase {pswd}"
         if "color" not in self.settings:
             self.settings["color"] = "#FF0000"
         if "stop_on_internet" not in self.settings:
             self.settings["stop_on_internet"] = False
         if "timeout_after_internet" not in self.settings:
             self.settings["timeout_after_internet"] = 90
+        self.wifi_command = "sudo wifi-connect --portal-ssid {ssid}"
```

Comment:
can you make this come settings also? that way downstream can replace it with a system service or a full path to the binary

Reply:
Makes sense; refactored to read from settings